### DISCUSSION TITLE
How to change image on twitter preview ?

### DISCUSSION THREAD

**Discussion by 0xARROWK** [thuliteio/doks#541] [Q&A] [answered]
Hello everyone !

Currently, the default image displayed on the link preview is a black image with "doks" in the center in white. The only image like this that I found in the treeview is located here :

```
├── static
      ...
│   ├── doks.png
      ...
```

I've obviously replaced it to show a custom preview, but it doesn't seem to affect the link preview. Do you know how to do this?

I should mention that the blog is hosted on netlify.

[EDIT]

I have saw in heads of netlify website (https://weam.netlify.app) that the url for twitter image card is always `doks.png` :

```html
<meta name="twitter:image" content="https://weam.netlify.app/doks.png">
```

However, I have deleted file `static/doks.png` and in my `config/_default/params.toml` file, I have set that :

```toml
schemaImage = "weam.png"
```

What I'm doing wrong ? Url for `https://weam.netlify.app/doks.png` return me a 404 but the image is always used as preview :(

Thanks for reading !

**Comment by h-enk** [comment, marked as answer]
Looks like you fixed it?

```html
 ..
<meta property="og:image" content="https://weam.netlify.app/weam.png">
..
<meta name=twitter:image content="https://weam.netlify.app/weam.png">
..
```

How did you manage to?

**Reply by 0xARROWK** [reply]
Hello ! It seems to have fixed itself, I haven't touched it since... I've just tested it on twitter and the image displays well. It must have been a cache problem on the netlify side.
